### ISSUE TITLE
sms received but no request made

### ISSUE THREAD

**Issue by h1ght1me** [sa3dany/android-sms-hooks#10]
debug logs
```
Received broadcast for SMS [ PaketKalan@25.10.2022 14:00 ]
Received broadcast for SMS [ PaketKalan@25.10.2022 14:06 ]
```

URL works & tested 
access to SMS is given on device 

does app supports http auth in URL form? (user:password@domain.com)?

**Comment by sa3dany** [comment]
Hi @h1ght1me, Thanks for the bug report. I haven't tested the app with HTTP basic auth URLs.
Please let me know your use case about using Basic Auth for the webhook URL so that I can consider supporting that in the future.

For now I've marked this a bug, because if credentials in the URLs are not supported, they should not be accepted as a webhook URL.

**Comment by h1ght1me** [comment]
> Hi @h1ght1me, Thanks for the bug report. I haven't tested the app with HTTP basic auth URLs. Please let me know your use case about using Basic Auth for the webhook URL so that I can consider supporting that in the future.
> 
> For now I've marked this a bug, because if credentials in the URLs are not supported, they should not be accepted as a webhook URL.

Yes, if URL is provided with basic auth, it's seems like ignored, because no errors was in debug logs
